src/Kernel/Magic/ViewMagic.cs

```diff
@@ -0,0 +1,130 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
```

Comment:
```suggestion
#nullable enable

```

Reply:
I think this was meant for `src/Jupyter/Visualization/ExecutionPathEncoder.cs`.